mlrun/config.py

```diff
@@ -101,6 +101,13 @@
     "artifact_path": "",  # default artifacts path/url
     # Add {{workflow.uid}} to artifact_path unless user specified a path manually
     "enrich_artifact_path_with_workflow_id": True,
+    "artifacts": {
+        # None is handled as True, reason is that we want the client to have precedence on both params bellow
+        # for example if the user logs a big artifact and the calculation get stuck for some reason, or it doesn't want
+        # the hash to be calculated so we wasn't the client to be able to configure that ( passed in client-spec )
```

Comment:
```suggestion
        # the hash to be calculated so we want the client to be able to configure that ( passed in client-spec )
```

The logic mentioned here is not fully implemented - see my comment in the `httpdb` class.